opengovsg/askgovsg
# feat(rte): add resize and remove image options

## Problem

The current image upload tool is unintuitive.

Closes #746 

## Solution

Provide options to resize and remove image appear upon clicking on the image, as per the [designs](https://www.figma.com/file/O6lxjjfn04Ow6nn2TMStpk/AskGov-Design-Master-v2?node-id=6343%3A84870). Postman's [code](https://github.com/opengovsg/postmangovsg) was also used as reference.

Note: edit alt text will be included in a separate PR

**Bug Fixes**:

- Expand the area that takes in drag and dropped images to match the grey box
- Add jpeg to accepted image upload formats (for documentation purposes - svg files are also accepted, however this has not been included as it's not as widely-used)
 
## Before & After Screenshots

**BEFORE**:

https://user-images.githubusercontent.com/56983748/142961108-77199674-5e29-47f0-bad7-935cdd7b5bd8.mov

**AFTER**:
Resize and remove image options:


https://user-images.githubusercontent.com/56983748/142960991-49841a57-6fbf-4c39-b20f-d16b8eb9dd7a.mov

Drag and drop area:

https://user-images.githubusercontent.com/56983748/142960724-7eadedb9-984d-4660-9cf1-88ce307a42fe.mov



## Tests

- Resize an image and check that the image size is reflected after submitting the post. The current image size should also be indicated by a disabled, darker button
- Remove image, check that image has been removed successfully
